api/blog/views.py

```diff
@@ -18,13 +18,19 @@ class TagViewSet(viewsets.ModelViewSet):
     serializer_class = TagSerializer
 
 
+class PostsPagination(pagination.LimitOffsetPagination):
+    default_limit = 10
+    max_limit = 50
```

Comment:
the values should be changed to:
```
default_limit = 4
max_limit = 10
```
We don't need to display too many blog posts at a time anywhere so these values should be fine.